serverless.yml

```diff
@@ -43,6 +43,7 @@ functions:
       LOOKUP_API_TOKEN: ${ssm:/housing-register/${self:provider.stage}/lookup-api-key}
       NEXT_PUBLIC_ENV: ${opt:stage}
       NEXT_PUBLIC_EVIDENCE_STORE: ${ssm:/housing-register/${self:provider.stage}/evidence-store-url}
+      NEXT_PUBLIC_GTM_ID: ${ssm:/housing-register/${self:provider.stage}/gtm-id}
```

Comment:
It'd be nice for a future project to understand if it's possible to import all the variables from a .env.sample file to save the need to duplicate them in here. for now, maybe should have a comment at the top of the sample file to say "don't forget to add your variables to serverless.yml too"